Utils/Dataflow/095_datasetInfoAMI/README

```diff
@@ -4,22 +4,16 @@
 
 1. Description
 --------------
-This stage is implemented to extract physics parameter values from AMI.
+Obtain physics parameters' values from AMI and append it to the supplied
+metadata of output datasets.
 
-
-2. Dataflow
------------
-
-093_datasetsFormat/output --> 095_datasetsFromAMI
-
-As we know, AMI maintains valid physics parameters only for datasets with the
-following data types/formats:
+AMI maintains valid physics parameters only for datasets with the following
+data types/formats:
    - AOD
    - EVNT
    - HITS
 
-So, during the datasets processing we filter datasetnames by data formats, and
-execute requests for AMI only for the filtered datasets. The rest of the
-records, which haven't pass the filter, we write to the output "AS-IS".
-It allows us to optimize time of data extraction from AMI, which is very
-time-consuming.
+Since requesting information from AMI is a very time-consuming process,
+the supplied datasets are filtered by data format - the ones with formats
+mentioned above are included into request to AMI and their metadata are
+updated, while all the others are left unchanged.
```

Comment:
Maybe it is worth mentioning that the request to AMI will also be executed in case of undefined data format (when `data_format` field of the input message is missed or empty).

Reply:
Done.